diffkemp/simpll/passes/ReduceFunctionMetadataPass.cpp

```diff
@@ -0,0 +1,25 @@
+//===----  ReduceFunctionMetadataPass.h - Removes some function metadata --===//
+//
+//       SimpLL - Program simplifier for analysis of semantic difference      //
+//
+// This file is published under Apache 2.0 license. See LICENSE for details.
+// Author: Tomas Glozar, tglozar@gmail.com
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file contains the declaration of the ReduceFunctionMetadataPass pass.
```

Comment:
'definition' instead of 'declaration' plus brief description of what the pass currently does (copy the description from below).